pkg/cmd/build/build.go

```diff
@@ -43,59 +43,43 @@ type BuildOptions struct {
 }
 
 // Run runs the build sequence
-func Run(ctx context.Context, buildOptions BuildOptions) error {
+func Run(ctx context.Context, namespace string, buildOptions BuildOptions) error {
```

Comment:
I think that `namespace` could be removed also from here if `buildWithOkteto` doesn't need it